src/Microsoft.ML.Transforms/ProjectionCatalog.cs

```diff
@@ -57,5 +57,48 @@ public static RandomFourierFeaturizingEstimator CreateRandomFourierFeatures(this
         /// <param name="columns"> Describes the parameters of the whitening process for each column pair.</param>
         public static VectorWhiteningEstimator VectorWhiten(this TransformsCatalog.ProjectionTransforms catalog, params VectorWhiteningTransformer.ColumnInfo[] columns)
             => new VectorWhiteningEstimator(CatalogUtils.GetEnvironment(catalog), columns);
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="LpNormalizingEstimator"/>.
```

Comment:
> Initializes a new instance of <see cref="LpNormalizingEstimator"/>. [](start = 11, length = 68)

this is a bad pattern for comments. Instead of implementation details, describe what the operation does. 
`Rescales the vector column to have unit norm` for example